src/rasenmaeher_api/db/people.py

```diff
@@ -173,6 +173,7 @@ async def list(cls, include_deleted: bool = False) -> AsyncGenerator["Person", N
                         Person.deleted == None  # pylint: disable=C0121 ; # "is None" will create invalid query
                     )
                 async for person in query.order_by(Person.callsign).gino.iterate():
+                    person.roles = [str(role) async for role in person.roles() if role is not None]
```

Comment:
I'm amazed that mypy did not complain about this. We can't just go adding undefined properties to the object that is supposedly well-defined.

The route should use a well defined pydantic schema and add the roles to that in the route